src/jdk.management/share/classes/com/sun/management/ThreadMXBean.java

```diff
@@ -121,7 +156,7 @@ public interface ThreadMXBean extends java.lang.management.ThreadMXBean {
      * </pre></blockquote>
      *
      * @return an approximation of the total memory allocated, in bytes, in
-     * heap memory for the current thread
+     * heap memory for the current thread,
```

Comment:
I am not sure if typos changes in the public API requires a CSR (albeit trivial one). Maybe skip these updates?

Reply:
Fixed.